minidump-stackwalk/stackwalker.cc

```diff
@@ -567,8 +567,9 @@ bool ConvertStackToJSON(const ProcessState& process_state,
       if (ContainsModule(modules_with_corrupt_symbols, frame->module)) {
         frame_data["corrupt_symbols"] = true;
       }
-      assert(!frame->module->code_file().empty());
-      frame_data["module"] = PathnameStripper::File(frame->module->code_file());
+      if (!frame->module->code_file().empty()) {
```

Comment:
In mozilla-central's minidump-analyzer I just store the empty string in this case. Either way works if downstream code assumes that a missing `module` means it's empty.

Reply:
Actually I was wrong, we don't store it at all, we just store the index so this is 100% fine.

Reply:
Thank you!